nameservice/tutorial/02-app-init.md

```diff
@@ -78,51 +55,9 @@ Here is what `baseapp` does:
 - Help initialise your state.
 - Help set up queries.
 
-Now you need to create a new custom type `nameServiceApp` for your application. This type will embed `baseapp` (embedding in Go similar to inheritance in other languages), meaning it will have access to all of `baseapp`'s methods.
-
-```go
-const appName = "nameservice"
-
-var (
-	// default home directories for the application CLI
-	DefaultCLIHome = os.ExpandEnv("$HOME/.nscli")
-
-	// DefaultNodeHome sets the folder where the application data and configuration will be stored
-	DefaultNodeHome = os.ExpandEnv("$HOME/.nsd")
-
-	// NewBasicManager is in charge of setting up basic module elements
-	ModuleBasics = module.NewBasicManager()
-
-	// account permissions
-	maccPerms = map[string][]string{}
-)
-
-type nameServiceApp struct {
-    *bam.BaseApp
-}
-```
-
-Add a simple constructor for your application:
-
-```go
-func NewNameServiceApp(logger log.Logger, db dbm.DB) *nameServiceApp {
-
-    // First define the top level codec that will be shared by the different modules. Note: Codec will be explained later
-    cdc := MakeCodec()
-
-    // BaseApp handles interactions with Tendermint through the ABCI protocol
-    bApp := bam.NewBaseApp(appName, logger, db, auth.DefaultTxDecoder(cdc))
-
-    var app = &nameServiceApp{
-        BaseApp: bApp,
-        cdc:     cdc,
-    }
-
-    return app
-}
-```
+Now you need to rename the `appName` & `newApp` types to the name of your app in this case you can use `nameservice` & `nameServiceApp` for your application. This type will embed `baseapp` (embedding in Go similar to inheritance in other languages), meaning it will have access to all of `baseapp`'s methods.
```

Comment:
```suggestion
Now you need to rename the `appName` & `newApp` types to the name of your app. In this case you can use `nameservice` & `nameServiceApp`. This type will embed `baseapp` (embedding in Go similar to inheritance in other languages), meaning it will have access to all of `baseapp`'s methods.
```